docs/quick-start.md

```diff
@@ -132,4 +163,6 @@ production.
lIGFyZSBhdCB0aGUgTWluaXN0cnk6IGhlIGVuZHMgdXAgbGVhZGluZyB0aGUgQXVyb3IgZGVwYXJ0bWVudC4gUm9uIGhlbHBzIEdlb3JnZSBhdCB0aGUgam9rZSBzaG9wIGFuZCBkb2VzIHZlcnkgd2VsbC4gR2lubnkgYmVjb21lcyBhIHByb2Zlc3Npb25hbCBRdWlkZGl0Y2ggcGxheWVyIGFuZCB0aGVuIHNwb3J0c3dyaXRlciBmb3IgdGhlIERhaWx5IFByb3BoZXQuPC9wPlxuICAgICAgICAgIDxwPjxhIGhyZWY9XCJodHRwczovL3d3dy5wb3R0ZXJtb3JlLmNvbS9jb2xsZWN0aW9uL2NoYXJhY3RlcnNcIiB0YXJnZXQ9XCJibGFua1wiPlJlYWQgbW9yZSBhYm91dCB0aGUgY2hhcmFjdGVyczwvYT48L3A-XG4gICAgICAgIDwvcGYtYWNjb3JkaW9uLXBhbmVsPlxuICAgICAgPC9wZi1hY2NvcmRpb24-XG4gICAgPC9zZWN0aW9uPlxuICAgIDxzZWN0aW9uPlxuICAgICAgPGgyPlRhYnMgY29tcG9uZW50PC9oMj5cbiAgICAgIDxwZi10YWJzPlxuICAgICAgICA8cGYtdGFiIHNsb3Q9XCJ0YWJcIiBpZD1cInRhYjFcIj5cbiAgICAgICAgICA8aDM-VGFiIDE8L2gzPlxuICAgICAgICA8L3BmLXRhYj5cbiAgICAgICAgPHBmLXRhYi1wYW5lbD5cbiAgICAgICAgICA8aDM-Q29udGVudCAxPC9oMz5cbiAgICAgICAgICA8cD5Mb3JlbSBpcHN1bSBkb2xvciBzaXQgYW1ldCwgPGEgaHJlZj1cIiNcIj5saW5rPC9hPiBjb25zZWN0ZXR1ciBhZGlwaXNpY2luZyBlbGl0LCBzZWQgZG8gZWl1c21vZCB0ZW1wb3IgaW5jaWRpZHVudCB1dCBsYWJvcmUgZXQgZG9sb3JlIG1hZ25hIGFsaXF1YS4gVXQgZW5pbSBhZCBtaW5pbSB2ZW5pYW0sIHF1aXMgbm9zdHJ1ZCBleGVyY2l0YXRpb24gdWxsYW1jbyBsYWJvcmlzIG5pc2kgdXQgYWxpcXVpcCBleCBlYSBjb21tb2RvIGNvbnNlcXVhdC4gRHVpcyBhdXRlIGlydXJlIGRvbG9yIGluIHJlcHJlaGVuZGVyaXQgaW4gdm9sdXB0YXRlIHZlbGl0IGVzc2UgY2lsbHVtIGRvbG9yZSBldSBmdWdpYXQgbnVsbGEgcGFyaWF0dXIuIEV4Y2VwdGV1ciBzaW50IG9jY2FlY2F0IGN1cGlkYXRhdCBub24gcHJvaWRlbnQsIHN1bnQgaW4gY3VscGEgcXVpIG9mZmljaWEgZGVzZXJ1bnQgbW9sbGl0IGFuaW0gaWQgZXN0IGxhYm9ydW0uPC9wPlxuICAgICAgICA8L3BmLXRhYi1wYW5lbD5cbiAgICAgICAgPHBmLXRhYiByb2xlPVwiaGVhZGluZ1wiIHNsb3Q9XCJ0YWJcIiBpZD1cInRhYjJcIj5cbiAgICAgICAgICA8aDM-VGFiIDI8L2gzPlxuICAgICAgICA8L3BmLXRhYj5cbiAgICAgICAgPHBmLXRhYi1wYW5lbD5cbiAgICAgICAgICA8aDM-Q29udGVudCAyPC9oMz5cbiAgICAgICAgICA8cD5Mb3JlbSBpcHN1bSBkb2xvciwgc2l0IGFtZXQgY29uc2VjdGV0dXIgYWRpcGlzaWNpbmcgZWxpdC4gT2ZmaWNpaXMgZWl1cyBpbGx1bSwgdXQgdmVsIHF1aXMgcG9ycm8gdm9sdXB0YXR1bSBhbWV0ISBFbmltLCBzZXF1aS4gTGF1ZGFudGl1bSBtYWduYW0gb2ZmaWNpYSBkb2xvcmUgZGViaXRpcyBxdWFzIGVpdXMgcGxhY2VhdCBiZWF0YWUgaWxsbyBvYmNhZWNhdGk_PC9wPlxuICAgICAgICA8L3BmLXRhYi1wYW5lbD5cbiAgICAgICAgPHBmLXRhYiByb2xlPVwiaGVhZGluZ1wiIHNsb3Q9XCJ0YWJcIiBpZD1cInRhYjNcIj5cbiAgICAgICAgICA8aDM-VGFiIDM8L2gzPlxuICAgICAgICA8L3BmLXRhYj5cbiAgICAgICAgPHBmLXRhYi1wYW5lbD5cbiAgICAgICAgICA8aDM-Q29udGVudCAzPC9oMz5cbiAgICAgICAgICA8cD5Mb3JlbSBpcHN1bSBkb2xvciBzaXQgYW1ldCBjb25zZWN0ZXR1ciwgYWRpcGlzaWNpbmcgZWxpdC4gRWEgYW5pbWkgZW5pbSBldW0gaXBzdW0gbGF1ZGFudGl1bSBlaXVzIGRlbGVuaXRpIHF1b3MgaWxsbyBpZCB2ZXJpdGF0aXMuIFZlcm8gdmVyaXRhdGlzIGFyY2hpdGVjdG8gYWxpcXVhbSBub24gdm9sdXB0YXRlIHF1aWJ1c2RhbSBzYWVwZSBpbiBjdW0uPC9wPlxuICAgICAgICA8L3BmLXRhYi1wYW5lbD5cbiAgICAgIDwvcGYtdGFicz5cbiAgICA8L3NlY3Rpb24-XG4gIDwvbWFpbj5cbjwvYm9keT5cbjwvaHRtbD4ifSx7Im5hbWUiOiJwYWNrYWdlLmpzb24iLCJjb250ZW50Ijoie1xuICBcImRlcGVuZGVuY2llc1wiOiB7XG4gICAgXCJsaXRcIjogXCJeMi4wLjBcIixcbiAgICBcIkBsaXQvcmVhY3RpdmUtZWxlbWVudFwiOiBcIl4xLjAuMFwiLFxuICAgIFwibGl0LWVsZW1lbnRcIjogXCJeMy4wLjBcIixcbiAgICBcImxpdC1odG1sXCI6IFwiXjIuMC4wXCJcbiAgfVxufSIsImhpZGRlbiI6dHJ1ZX1d) example of the HTML above
```

Comment:
This should be a gist url

Reply:
> This should be a gist url

Is there a way to create a gist under the org instead of being tied to a user?

Reply:
> Is there a way to create a gist under the org instead of being tied to a use

Couldn't find a way to do this or if it is even possible, just exists under my user for now.